src/twisted/newsfragments/9660.bugfix

```diff
@@ -0,0 +1 @@
+twisted.internet._glibbase.PortableGlibReactorBase.simulate no longer raises TypeError when there are no delayed called. This was a regression introduced with the migration to Python3 in which the builtin `min` function no longer accepts `None` as an argument.
```

Comment:
True. My bad.

The tests failed ... most probably due to a flaky tests... so it needs a re-run.

```suggestion
twisted.internet.gireactor.PortableGIReactor.simulate and twisted.internet.gtk2reactor.PortableGtkReactor.simulate no longer raises TypeError when there are no delayed called. This was a regression introduced with the migration to Python3 in which the builtin `min` function no longer accepts `None` as an argument.
```